src/Streamly/Internal/Data/Array/Stream/Foreign.hs

```diff
@@ -23,16 +23,16 @@ module Streamly.Internal.Data.Array.Stream.Foreign
 
     -- * Elimination
     -- ** Element Folds
-    , fold
-    , parse
-    , parseD
+    , foldBreak
+    , parseBreak
+    , parseBreakD
 
     -- ** Array Folds
-    , foldArr
-    , foldArr_
+    , arrayFold
+    , arrayFoldBreak
```

Comment:
Because we have a `fold` operation as well in the same module `foldArrays` would be confusing as both operations are folding arrays, then one may ask what is the difference. The idea was that `arrayFold` sounds like `runArrayFold`, but we can explicitly name it that.